src/bin/restful-react-import.ts

```diff
@@ -0,0 +1,93 @@
+import program from "commander";
+import { existsSync, readFileSync, writeFileSync } from "fs";
+import inquirer from "inquirer";
+import { join, parse } from "path";
+import request from "request";
+import importOpenApi from "../scripts/import-open-api";
+
+program.option("-o, --output [value]", "output file destination");
+program.option("-f, --file [value]", "input file (yaml or json openapi specs)");
+program.option("-g, --github [value]", "github path (format: `owner:repo:branch:path`)");
+program.parse(process.argv);
+
+(async () => {
+  if (program.file) {
+    const data = readFileSync(join(process.cwd(), program.file), "utf-8");
```

Comment:
What happens if this fails?

Reply:
![image](https://user-images.githubusercontent.com/1761469/49381089-75f17d80-f713-11e8-9d0c-96ebcd5663f7.png)

This, to have a more explicit response ;)